File: algorithms/strings/reverse_string/reverse_string.py
def reverse_string(string_arg):
    strLength = len(string_arg)
    newString = string_arg[strLength-1]

    for i in range(strLength-2,-1,-1):
        newString += string_arg[i]
    return newString


arg_string = "Hello World"
print(reverse_string(arg_string))

File: algorithms/strings/reverse_string/reverse_string.js
function reverse_string(arg_string) {

  var strLength = arg_string.length;
  var newString = arg_string[strLength-1];

  for (let i = strLength-2; i >= 0; i--) {
    newString += arg_string[i];
  }
  return newString;
}


var arg_string = "Hello World";

console.log(reverse_string(arg_string));
